docs/reference/ml/trained-models/apis/delete-trained-models.asciidoc

```diff
@@ -18,8 +18,10 @@ Deletes an existing trained {infer} model.
 [[ml-delete-trained-models-prereq]]
 == {api-prereq-title}
 
-Requires the `manage_ml` cluster privilege. This privilege is included in the 
+* Requires the `manage_ml` cluster privilege. This privilege is included in the
 `machine_learning_admin` built-in role.
+* If the {es} {security-features} are enabled, you must have the `monitor` or `manage`, and
```

Comment:
Maybe it makes more sense to just link directly to the task apis? That way we don't have to keep these in sync

https://www.elastic.co/guide/en/elasticsearch/reference/current/tasks.html

Reply:
I talked to @droberts195 and this is no longer applicable. We'll use a client with the ml origin instead to avoid the privilege issues.